module-3/cdk/.gitignore

```diff
@@ -0,0 +1,7 @@
+*.js
```

Comment:
I wonder if it makes sense that each module will be it's own CDK app. Why not just create a single CDK app for the entire thing and just add stacks to it? Will save a lot of set up and ceremony.